tests/regression/gdoc/test_chrome_gdoc_create_isolate_image_1.py

```diff
@@ -5,7 +5,7 @@ class TestSikuli(PerfBaseTest):
 
     def setUp(self):
         self.pre_run_script = "test_chrome_gdoc_read_blank_page"
-        self.pre_run_script_test_url_id = "TEST_TARGET_ID_BLANK_PAGE"
+        self.pre_run_script_test_url_id = self.global_config['gsuite']['gdoc-blank-page']
```

Comment:
I think create_isolate_image_1 and create_image_1 are the same test case. So do create_isloate_table_1 and create_isolate_table_1. We should keep only one of them.

Also, pre_run_script was taken out in #596, and we probably should use the new `self.set_variable(test_target=self.global_config['gsuite']['gdoc-blank-page'])`

@MikeLien @ShakoHo @askeing, thoughts?